.github/workflows/validate-pr.yml

```diff
@@ -0,0 +1,38 @@
+name: validate-pr
+
+on:
+  pull_request:
+    types: [opened, edited, labeled, unlabeled]
+
+jobs:
+  check-changelog:
+    if: contains(github.event.pull_request.labels.*.name, 'impact/changelog')
+    runs-on: ubuntu-20.04
+    env:
+      PR_BODY: |
+        ${{ github.event.pull_request.body }}
+    steps:
+      - name: Check changelog description
+        run: |
+          # Extract everything after the Description for the changelog section
+          section=$(echo -n "$PR_BODY" | awk '/\*\*- Description for the changelog\*\*/,EOF')
+
+          # Extract the release-note code block
+          block=$(echo -n "$section" | awk '/```release-note/{flag=1;next}/```/{flag=0}flag')
```

Comment:
This gives me https://stackoverflow.com/a/1732454/1811501 feels 🙈 wondering if just looking for the ```release-note` section would work 🤔 


I should add that I've deliberately used `markdown` as code-hint for these, so that I could create a release-note with Markdown syntax (as they would be used), which would also highlight the block correctly.

Wondering if it would support custom annotations / options, like https://github.github.com/gfm/#example-113

````
```markdown release-notes
- hello world [foo](https://example.org}
```
````

It looks like it doesn't "barf" at least

```markdown release-notes
- hello world [foo](https://example.org}
```